client/components/deposits-overview/recent-deposits.tsx

```diff
@@ -0,0 +1,122 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { Flex, FlexItem, Icon } from '@wordpress/components';
+import { calendar } from '@wordpress/icons';
+
+/**
+ * Internal dependencies.
+ */
+import strings from './strings';
+import './style.scss';
+import DepositStatusChip from 'components/deposit-status-chip';
+import { getDepositDate } from 'deposits/utils';
+import { CachedDeposit } from 'wcpay/types/deposits';
+import { useDeposits } from 'wcpay/data';
+import { formatCurrency } from 'wcpay/utils/currency';
+import { Link } from '@woocommerce/components';
+import { getDetailsURL } from 'wcpay/components/details-link';
+
+interface DepositRowProps {
+	deposit: CachedDeposit;
+}
+interface RecentDeposits {
+	deposits: CachedDeposit[];
+	isLoading: boolean;
+}
+interface RecentDepositsProps {
+	currency: string | undefined;
+}
+
+const tableClass = 'wcpay-deposits-overview__table';
+
+const GetRecentDeposits = ( currency?: string ): RecentDeposits => {
+	const query = {
+		status_is_not: 'estimated',
+		store_currency_is: currency,
+		orderby: 'date',
+		order: 'desc',
+		per_page: '3',
+	};
+	const deposits = useDeposits( query );
+
+	return {
+		deposits: deposits.deposits,
+		isLoading: deposits.isLoading,
+	};
+};
+
+const DepositTableRow: React.FC< DepositRowProps > = ( {
+	deposit,
+}: DepositRowProps ): JSX.Element => {
```

Comment:
Nit: `DepositRowProps` isn't also required here, as it's passed to ` React.FC< DepositRowProps >`

```suggestion
} ): JSX.Element => {
```